src/main/resources/org/jenkinsci/plugins/pipeline/utility/steps/conf/WriteYamlStep/help.html

```diff
@@ -34,7 +34,11 @@
     </li>
     <li>
         <code>data</code>:
-        A Mandatory Object containing the data to be serialized.
+        An Optional Object containing the data to be serialized.
+    </li>
+    <li>
```

Comment:
It should be mentioned here that it's either `data` or `datas` that can be specified.

Reply:
Done